svc/FetchService.js

```diff
@@ -117,13 +121,70 @@ export class FetchService {
      *     If not explicitly set in opts then the contentType will be set based on the method. POST
      *     requests will use 'application/x-www-form-urlencoded', otherwise 'text/plain' will be
      *     used.
+     * @param {Object} [opts.qsOpts] - Object of options to pass to the param converter, qs.
```

Comment:
Why are we repeating all this parameter documentation for every fetchJSON/postJSON/putJSON method?  Very difficult to maintain, and we already note that the options bundle is a bundle to "pass through to fetch().  Can we only document the new parameters for each method, and have the docs provide a direct {link} back to the fetch method?